source/clear-linux/tutorials/gce/gce.rst

```diff
@@ -0,0 +1,126 @@
+.. _clear-on-gce:
+
+Launch |CL-ATTR| Compute Engine on Google Cloud Platform
+########################################################
+
+|CL-ATTR| is a Linux distribution designed for bare mental and virtual machines
+running from the cloud to the edge. This tutorial walks you through the steps
+to create virtual machines in
+`Google Cloud Platform (GCP) <https://cloud.google.com/>`_ running |CL|.
+Reference :ref:`azure` and :ref:`aws-web` for launching |CL| VMs on
+Microsoft\* Azure\* cloud services and Amazon\* Web Services respectively.
+
+#. Google Cloud Platform uses **Projects** to manage resources. Sign in
+   your Google account on the
+   `Google Cloud Console <https://console.cloud.google.com/>`_
+   , and select or create a new project.
+   Reference the
+   `Quickstart Using a Linux VM <https://cloud.google.com/compute/docs/quickstart-linux>`_
+   guide to get familiar with the process of creating VM instances in GCP.
+
+   .. figure:: figures/00-sign-in.png
+      :scale: 50 %
+      :alt: Sign in Google services
+
+#. Download the |CL| image archive for Google Compute Engine (GCE)
+   *clear-<release number>-gce.tar.gz*
+   from the |CL-PRJ| image repository. You don't need to uncompress the image
```

Comment:
Remove " from the |CL-PRJ| image repository."

Reply:
removed